traefik/templates/poddisruptionbudget.yaml

```diff
@@ -0,0 +1,17 @@
+{{- if .Values.podDisruptionBudget -}}
```

Comment:
Would recommend an explicit `podDisruptionBudget.enabled` field so users can turn it on/off for different environments without removing default config for min/max

Reply:
Done